### PR TITLE
Deployment for Kovan

### PR DESCRIPTION
**Pull request by josojo**
This PR includes:

cleaning for the audit.
new fee structure
new OWL burning mechanism

The following function names have been changed:
getPriceForJS=>getPriceExt
getPriceOracleForJS => priceOracleExt
computeRatioOfHistoricalPriceOraclesForJS=>computeRatioOfHistoricalPriceOraclesExt

scripts to inject/extract addresses of deployments in folder scripts

network.json with the deployed contracts in the kovan network.

In order to talk to the kovan or rinkeby network:
=>truffle compile -all
=> node scripts/inject_network_info.js
=> talk to contracts with ContractName.deployed(), network Id should be detected automatically

like this
josojo@josojo-XPS-13-9360:~/gnosis/dutch-exchange$ truffle console --network kovan
truffle(kovan)> PriceOracleInterface.deployed().then((p)=>console.log(p.address))
0x183a79a1a8d487ef6ebbf23704a10cfe54b36298
undefined
truffle(kovan)> 
(To exit, press ^C again or type .exit)
truffle(kovan)> 
josojo@josojo-XPS-13-9360:~/gnosis/dutch-exchange$ truffle console --network development
truffle(development)> PriceOracleInterface.deployed().then((p)=>console.log(p.address))
0x0290fb167208af455bb137780163b7b7a9a10c16

### CODE REVIEW

**Review by anxolin — COMMENTED** [review]
Cool, thanks for the heads up, we'll adapt to the new method names as soon as it's merged.

File: contracts/.solhint.json [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{                                              
+diud{
```
Comment (Velenir) [inline contracts/.solhint.json:1]: typo ~diud~

**Comment by W3stside** [comment]
@josojo can this be closed?

**Comment by W3stside** [comment]
@josojo can this be closed?

**Pull request closed without merging**